docs/05-PubsubForValidators.md

```diff
@@ -0,0 +1,58 @@
+# The x/pubsub Module for Validators
+
+Before Sommelier v7, the Steward Registry was used to record the CA and server endpoint of all steward processes. This database was then used by Strategy Providers (SPs) to send requests to each steward instance. 
+
+The Steward Registry has been deprecated in favor of a (mostly) generalized pubslisher/subscriber registry Cosmos SDK module, [x/pubsub](https://github.com/peggyjv/sommelier/tree/main/x/pubsub).
+
+For more information on x/pubsub itself, please refer to the [protocol documentation]()
```

Comment:
Missing link here as well.

Reply:
I don't think this exists yet, I had the gitbook pubsub page in mind. Does it?

Reply:
Hrmm I suppose not. Will have to do that.